README.md

```diff
@@ -67,10 +67,10 @@ counts.writeAsCsv(outputPath)
 
 Prerequisites for building Flink:
 
-* Unix-like environment (We use Linux, Mac OS X, Cygwin)
+* Unix-like environment (we use Linux, Mac OS X, Cygwin)
 * git
 * Maven (we recommend version 3.2.5)
-* Java 8
+* Java 8 (exactly 8, not 9 or 10)
```

Comment:
Can we phrase this more like "(Java 9 and 10 are not yet supported)

Reply:
sure, and done :-)